src/sentry/static/sentry/app/views/events/searchBar.tsx

```diff
@@ -25,25 +26,34 @@ const FIELD_TAGS = Object.fromEntries(
   Object.keys(FIELDS).map(item => [item, {key: item, name: item}])
 );
 
-class SearchBar extends React.PureComponent {
-  static propTypes = {
+type SearchBarProps = Omit<React.ComponentProps<typeof SmartSearchBar>, 'tags'> & {
+  api: Client;
+  organization: Organization;
+  tags: {[key: string]: Tag};
+  omitTags?: string[];
+  projectIds?: number[] | Readonly<number[]>;
+  fields?: Readonly<Field[]>;
+};
+
+class SearchBar extends React.PureComponent<SearchBarProps> {
+  static propTypes: any = {
     api: PropTypes.object,
-    organization: SentryTypes.Organization,
-    tags: PropTypes.objectOf(SentryTypes.Tag),
-    omitTags: PropTypes.arrayOf(PropTypes.string),
-    projectIds: PropTypes.arrayOf(PropTypes.number),
-    fields: PropTypes.arrayOf(PropTypes.object),
+    organization: PropTypes.object,
+    tags: PropTypes.object,
```

Comment:
Curious about why we need these changes?

Reply:
You mean for `tags`? Are  propTypes and Types compatible? 🤔  maybe I misread a type error while playing with it.

Reply:
For tags and org, we should be able to keep using their respective `SentryTypes`.

Reply:
Ah I must have messed something up then, I'll give that a shot, thanks 👍